morphocut/str.py

```diff
@@ -28,7 +31,9 @@ class ParseWarning(UserWarning):
 
 @Output("meta")
 class Parse(Node):
-    """Parse information from a path.
+    """Parse information from a path. The class can be used to simply parse a string
+    Or to search a string in some pattern.
+    It also uses external library 'parse'
```

Comment:
I can only repeat myself: Do it just like I did in `morphocut.pims.BioformatsReader`. Use pydocstyle to get feedback on your docstrings.

- https://trello.com/c/ub9i0YrA/14-write-a-docstring
- https://morphocut.readthedocs.io/en/latest/contributing.html#code-style

The nature of the string is irrelevant here. Don't mention paths. Write a proper `note` and link to the documentation of `parse`.